coding/javascript.md

```diff
@@ -91,6 +111,9 @@ If you'd like introduce a new library, or feel we should replace one of the abov
 
 #### Reducers
 
+* Ensure you use [immutable patterns](https://redux.js.org/recipes/structuringreducers/immutableupdatepatterns) for updating the redux store. Using
```

Comment:
This and the item below are duplicated now :)

Reply:
thanks!